Formula/manticoresearch.rb

```diff
@@ -6,20 +6,24 @@ class Manticoresearch < Formula
   version_scheme 1
   head "https://github.com/manticoresoftware/manticoresearch.git", branch: "master"
 
-  # NOTE: Do not update to odd patch version (e.g. 5.0.3)
   stable do
     url "https://github.com/manticoresoftware/manticoresearch/archive/refs/tags/5.0.2.tar.gz"
     sha256 "ca7828a6841ed8bdbc330516f85ad3a85749998f443b9de319cec60e12c64c07"
 
     # Allow system ICU usage and tune build (config from homebrew; release build; don't split symbols).
     # Remove with next release
-
     patch do
       url "https://github.com/manticoresoftware/manticoresearch/commit/70ede046a1ed.patch?full_index=1"
       sha256 "8c15dc5373898c2788cea5c930c4301b9a21d8dc35d22a1bbb591ddcf94cf7ff"
     end
   end
 
+  # Only even patch versions are stable releases
+  livecheck do
+    url :stable
+    regex(/^v?(\d+(?:\.\d+)*\.\d*[02468])$/i)
```

Comment:
```suggestion
    regex(/^v?(\d+(?:\.\d+)+\.\d*[02468])$/i)
```

This repository has consistently used a three-part version scheme like 1.2.3 but this regex would technically match something like 1.2 while not matching something like 1.3. I don't think this quite represents the intention in terms of enforcing an even-numbered patch version, so we should use `(?:\.\d+)+` or `\.\d+` (instead of the optional `(?:\.\d+)*`) to account for the minor version.

It's not an issue with the current tags but I've seen repositories do unexpected things, so better safe than sorry.